Dockerfile

```diff
@@ -62,6 +62,20 @@ COPY config config
 COPY devtools devtools
 RUN npm install
 
+# Fetch external Bazel artifacts.
+# Copy over the WORKSPACE file, everything it imports, and bazelisk.
+COPY tools/bazelisk* tools/
+COPY build_defs/emscripten build_defs/emscripten
+COPY build_defs/kotlin_native build_defs/kotlin_native
```

Comment:
Maybe wait a bit until I can land my platform dependency change?

Reply:
Yes, we can wait for your PR to land.